src/accordion.styles.ts

```diff
@@ -12,6 +12,17 @@ export default [
       border-radius: var(--glide-core-rounding-base-radius-md);
       box-shadow: var(--glide-core-effect-lifted);
       font-family: var(--glide-core-typography-family-primary);
+
+      &.prefix-icon {
+        .default-slot {
+          padding-inline-start: 3.25rem;
```

Comment:
Where we can, we try not to style one element via another so that each style declaration for an element is the source of truth for all its styles. 

So, instead of styling `.default-slot` from `.component`, I'd recommend applying this style directly to `.default-slot` via a conditional class similar to `.indented`.

Reply:
@clintcs - Makes sense! So the `.prefix-icon-slot` class already existed, but wasn't being used for anything. Would you prefer I:

1. Change the existing `.prefix-icon-slot` to be conditional, only added if there is a prefix icon
2. Keep the existing `.prefix-icon-slot` as-is, and add a conditional `.has-prefix-icon` class
3. Remove the existing `.prefix-icon-slot`, and only have a conditional `.has-prefix-icon` class
4. Something else.. :)

Reply:
Spoke offline: we went with option 2, except changing the name of the new class to `.slotted-content`